### PR TITLE
Wrote additional tests for the About Us page Press section

### PR DESCRIPTION
**Pull request by TimeaKovacs86**
…ommand to commands.js check link with Xpath, added new social contact (pr) to socialContactInfo.json

### What does this pull request aim to achieve? 💯

Created additional tests for the Press section on the About Us page

- [ ] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

**Comment by TimeaKovacs86** [comment]
Hi everyone! As you can see I added tests to check if the press section is working well on the About Us page. For now I could have reach certain elements just by the xpath cause they don't have higher level selectors, when that will change we can update these tests as well. Until that Please review! Let me know if there are any suggestions.

**Comment by adorapham** [comment]
@TimeaKovacs86 , I run locally and get 2 failed, do you get the same or all passes? TY
image

**Comment by adorapham** [comment]
Hi TimeaKovacs86  , I run and get 3 failing 
image

**Comment by galeza** [comment]
Hi @TimeaKovacs86 
There are some tests failing:
1. We change Instagram account. Now it's https://www.instagram.com/fightpandemicshq/
2.   
```
    xpath //*[@id=\"root\"]/div/div/div[3]/main/div/div[8]/div[1],
    assert expected **{ Object (length) }** to be **visible**,
```

3. Problem with retrieving Go to Press button or link?

File: cypresse2e/cypress/elements/pages/aboutUs.js [hunk 1]
```diff
@@ -13,6 +13,8 @@ class AboutUs {
   joinStudentProgramLink = "https://apply.workable.com/fightpandemics/j/58B157AAB2/";
   getToHelpBoardButton = "#AU_VHB";
   goToHelpBoardLink = "/feed";
+  getToPressKitButton = "//*[@id=\"root\"]/div/div/div[3]/main/div/div[9]/div/a";
```
Comment (galeza) [inline cypresse2e/cypress/elements/pages/aboutUs.js]: Could you try this CSS selector: div[direction="column"] > a[role="button"] or div[direction="column"] > a[role="button"] span

File: cypresse2e/cypress/elements/pages/aboutUs.js [hunk 2]
```diff
@@ -13,6 +13,8 @@ class AboutUs {
...
+  goToPressKitLink = "https://www.notion.so/fightpandemics/Press-Kit-7146b85a49c848ec8395c0cadf3371b5";
   lifetimeSupportersContainerXpath = "//*[@id=\"root\"]/div/div/div[3]/main/div/div[8]/div[1]";
```
Comment (galeza) [inline cypresse2e/cypress/elements/pages/aboutUs.js]: Maybe we can use CSS Selector: 'main > div > div:nth-child(9) > div:nth-child(4)'
I think that we can also ask for adding IDs on the About Us page or do this on our own.

File: cypresse2e/cypress/fixtures/socialContactInfo.json [hunk 3]
```diff
@@ -4,7 +4,9 @@
   "instagramLink": "https://www.instagram.com/fightpandemics/",
```
Comment (galeza) [inline cypresse2e/cypress/fixtures/socialContactInfo.json]: This was changed to: https://www.instagram.com/fightpandemicshq/

**Review by galeza — CHANGES_REQUESTED** [review]

**Comment by galeza** [comment]
@Zaydme 
May I add some IDs in the AboutUs.js file in order to simplify our tests?
I mean for these containers: SupportersLogosContainer (h3 lifetimeSupporters, h3 currentSupporters, h3 pastSupporters) and PressContainer a.

**Or maybe we should add GTM tags here?**

**Comment by Zaydme** [comment]
@galeza You can add ids to any non-clickable elements such as `p` or `h3` etc.. 
But for a clickable element, you can put an id as long as it doesn't already have a GTM id, or doesn't need one.
For the PressContainer a, I see that other anchor elements in the same page with emails (`href="mailto:*"`) don't have a GTM id, so I suppose this one doesn't need one too.

**Comment by TimeaKovacs86** [comment]
Alright, all of my tests passed last night when I committed this pull request, so I think my localhost was cashed.. I see now the failing tests, I will do the changes. And since we have a new "current community partner" as well, what we should test alongside with the supporter containers if it is there or not, I will write those tests selecting the elements with the class (which is the same in all of these containers), hopefully it will work this time.

**Review by jessica-lau — CHANGES_REQUESTED** [review]
Encountered 3 failed tests:

image

The option to add ids as @galeza suggested would certainly help make it easier.

**Comment by TimeaKovacs86** [comment]
Normally now this should pass with flying colors!
Cypress About Us Page Update tests pass
Docker About US page Update tests pass

**Review by jessica-lau — DISMISSED** [review]
All tests passed. LGTM for now until potential ids are added down the road. Thanks @TimeaKovacs86 

image

**Comment by adorapham** [comment]
LGTM, TY @TimeaKovacs86 
image

**Review by adorapham — DISMISSED** [review]

**Comment by TimeaKovacs86** [comment]
Thank you for your help girls! If you find it good as well, please merge @galeza!

**Comment by galeza** [comment]
@TimeaKovacs86 
Please check #2039

File: cypresse2e/cypress/elements/pages/aboutUs.js [hunk 4]
```diff
@@ -13,12 +13,16 @@ class AboutUs {
   joinStudentProgramLink = "https://apply.workable.com/fightpandemics/j/58B157AAB2/";
   getToHelpBoardButton = "#AU_VHB";
   goToHelpBoardLink = "/feed";
-  lifetimeSupportersContainerXpath = "//*[@id=\"root\"]/div/div/div[3]/main/div/div[8]/div[1]";
-  currentSupportersContainerXpath = "//*[@id=\"root\"]/div/div/div[3]/main/div/div[8]/div[2]";
-  pastSupportersContainerXpath = "//*[@id=\"root\"]/div/div/div[3]/main/div/div[8]/div[3]";
+  getToPressKitButton = "#root > div > div > div.am-drawer-content > main > div > div.sc-jOFreG.dCiTET > div > a";
```
Comment (Zaydme) [inline cypresse2e/cypress/elements/pages/aboutUs.js]: Those classNames (exp: `.sc-jOFreG.dCiTET`) are randomly generated for each build and are not consistent.

**Comment by jessica-lau** [comment]
@TimeaKovacs86 I'll re-review after you add the new ids that just merged. Thanks!

**Review by adorapham — COMMENTED** [review]
Put it back to pending after @Zaydme 's comment. that is good to know those classNames (exp: .sc-jOFreG.dCiTET) are randomly generated for each build and are not consistent.
@TimeaKovacs86 ,testing on staging after the ID merged and it works, the code is clean now 👍 . LGTM -TY
image

**Comment by TimeaKovacs86** [comment]
I did the suggested changes, hopefully this time I didn't miss anything. Please review and get back to me if something is wrong. Thank you very much!

**Comment by TimeaKovacs86** [comment]
BTW I run the tests for staging not in localhost.
About Us page with Press Kit tests update with IDs on staging

**Review by adorapham — DISMISSED** [review]

**Review by jessica-lau — COMMENTED** [review]
@TimeaKovacs86 All tests passed but I didn't realize that the community partners didn't have an id assigned to it when I reviewed #2039 so I created PR [2043](https://github.com/FightPandemics/FightPandemics/pull/2043) to add one. cc: @Zaydme @galeza

Alright, then I will wait fro that ID to be merged and then I will change this pull request by adding that ID to the community partners in this test.

That is good , next time we can add ID ourselves if needed.  currentCommunityPartnerXpath is not a xpath any more and supporterContainerVisibleXpath/getSupporterContainerXpath are no longer needed.  I will let my review status as it is because I know it will be passed 💯

**Comment by TimeaKovacs86** [comment]
Here we go!! Added the ID for the Community Partner Section and refactored the test! Please review! And thank you for all the help! Really appreciate it!

**Review by jessica-lau — APPROVED** [review]
LGTM. thanks @TimeaKovacs86

**Review by galeza — APPROVED** [review]
LGTM

**Comment by galeza** [comment]
@TimeaKovacs86 
Let me know if I can merge it

**Comment by TimeaKovacs86** [comment]
Thank you for the reviews!
@galeza  Please merge! TY

**Comment by adorapham** [comment]
LGTM , TY 
image

**Review by adorapham — APPROVED** [review]

**Pull request merged**